exercises/calc/calc.py

```diff
@@ -12,48 +13,54 @@
     srp1
 )
 
-
+# Define custom packet class for P4calc
 class P4calc(Packet):
     name = "P4calc"
-    fields_desc = [ StrFixedLenField("P", "P", length=1),
-                    StrFixedLenField("Four", "4", length=1),
-                    XByteField("version", 0x01),
-                    StrFixedLenField("op", "+", length=1),
-                    IntField("operand_a", 0),
-                    IntField("operand_b", 0),
-                    IntField("result", 0xDEADBABE)]
-
+    # Define fields for the P4calc packet
+    fields_desc = [ StrFixedLenField("P", "P", length=1), 
+                    StrFixedLenField("Four", "4", length=1),  
+                    XByteField("version", 0x01),  
+                    StrFixedLenField("op", "+", length=1),  
+                    IntField("operand_a", 0),  
+                    IntField("operand_b", 0),  
+                    IntField("result", 0xDEADBABE)]  
+
```

Comment:
You should only make a comment here. Why were you removing `fields_desc` and copying it again?